make/testing_flags.mk

```diff
@@ -81,11 +81,11 @@ LOG_TAGS := nolog
 endif
 
 # If a timeout was requested, construct initialize the proper flag for the go
-# test command. If not, we set 60m (up from the default 10m).
+# test command. If not, we set 120m (up from the default 10m).
```

Comment:
added some context. This value is actually increased again in the final PR, and some background here is, since our itest builds have been failing for years, we never know how long it takes to finish them all, until now. It took more than 3h on my local machine to run a `make itest`, and for the builds they vary from 1h30m to 2h30m, still fixing tho.